static/templates/settings/account-settings.handlebars

```diff
@@ -132,5 +132,48 @@
             </div>
         </div>
 
+        <hr class="settings_separator" />
+
+        <div class="form-horizontal" id="api_key_button_box">
+            <div class="input-group side-padded-container">
+                <p class="api-key-note">
+                    {{#tr this}}
+                    For most bots using the API, you'll want to give each bot
+                    its own name and API key using the above section.  But if you
+                    want to write a bot that can access your own private
+                    messages, you should use your personal API key.
+                    {{/tr}}
```

Comment:
@timabbott what about "For most bots using the API, you'll want to give each bot its own name and API key.  But if you want to write a bot that can access your own private messages, you should use your personal API key." ? (basically removing "using the above section")